app/src/main/java/woowacourse/paint/MainActivity.kt

```diff
@@ -1,14 +1,51 @@
 package woowacourse.paint
 
 import android.os.Bundle
+import android.view.View
+import android.view.View.OnClickListener
 import androidx.appcompat.app.AppCompatActivity
+import woowacourse.paint.databinding.ActivityMainBinding
 
-
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity(), OnClickListener {
+    private lateinit var viewBinding: ActivityMainBinding
+    private var thickness = 0f
+    private var color = 0
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        viewBinding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(viewBinding.root)
+        setListener()
+        viewBinding.rangebar.valueFrom = 0.0f
+        viewBinding.rangebar.valueTo = 100.0f
```

Comment:
```suggestion
android:valueFrom="0"
android:valueTo="100"
```
이 부분은 XML에서도 설정이 가능해요 ;)

Reply:
그런 방법도 있었군요! 감사합니다 🙃  
반영했고, 추가로 아래와 같은 속성도 있더군요!
```
android:stepSize="5"
```
이 속성을 설정해주면 Slider의 Step을 몇으로 할지 정할 수 있답니다!

Reply:
오,, 이런 속성은 몰랐네요!! 감사합니다 :)
이번 미션의 경우로 예를 들면, brush의 두께 크기를 일정한 간격의 단위로 정한 경우 유용한 설정이겠네요 ;)